src/core/elf.c

```diff
@@ -3,10 +3,54 @@
 #include <mmu/paging.h>
 #include <string.h>
 
+#define ELF_TYPE_EXECUTABLE         0x2
+
+#define ELF_PROGRAM_TYPE_LOAD       0x1
+
+#define ELF_FLAG_EXECUTE            0x1
+#define ELF_FLAG_READ               0x4
+#define ELF_FLAG_WRITE              0x2
+
+#define ELF_SECTION_INDEX_UNDEFINED 0 //Undefined or meaningless section reference
```

Comment:
Could you please change the comments so that they are Doxygen compliant and similar to other parts of the project?

```suggestion
/// Undefined or meaningless section reference.
#define ELF_SECTION_INDEX_UNDEFINED 0
```

Reply:
Done